browser/src/VariantList/Variants.tsx

```diff
@@ -164,15 +168,19 @@ const Variants = ({
   )
 
   const [positionLastClicked, setPositionLastClicked] = useState(null)
-  // @ts-expect-error TS(7006) FIXME: Parameter 'position' implicitly has an 'any' type.
-  const onNavigatorClick = useCallback((position) => {
+  const [termLastSearched, setTermLastSearched] = useState(null)
+
+  const createCallback = useCallback((sortByKey: string, stateSetter: any) => (position: number) => {
     setSortState({
-      sortKey: 'variant_id',
+      sortKey: sortByKey,
```

Comment:
It seems like this could be narrowed down by using the existing `Variant` type and getting all of its possible properties. I might ping Phil on this to see if he knows a way to do this, some brief searching didn't yield any good results.

Edit:
Then again, since users are sorting based on clicking the rendered column headers, they will by definition always be sorting by a valid `sortKey`, so maybe stronger typing here doesn't actually aid in safety of the code.